### DISCUSSION TITLE
Release of Jedis with unified binary commands interface?

### DISCUSSION THREAD

**Discussion by jbank** [xetorthio/jedis#2556] [Q&A] [answered]
Hello, I couldn't find a better way of getting this information.

I saw that this pull request went in (https://github.com/redis/jedis/pull/2533) and I was wondering what is the timeline for a release including this? This would make our codebase significantly simpler to support standalone, sharded and clustered Redis with the same interface.

Waiting for the next release with bated breath. So glad to see this happening!

Thanks

**Comment by sazzad16** [comment, marked as answer]
Hi @jbank, it's great to hear that #2533 is (likely to be) welcomed by the users.

Jedis snapshots are being published as well. Is using those an option for you?

**Reply by jbank** [reply]
ok I tried to use the snapshots but it seems like the repository url doesn't work anymore. 
https://oss.sonatype.org/content/repositories/snapshots

I am assuming that I am doing something obviously wrong but I can't tell what that would be. Any advice would be appreciated.

**Reply by sazzad16** [reply]
May I see your pom? Please hide any personal/critical info but keep the whole structure.

**Comment by jbank** [comment]
I think that snapshots should be good in the short term. I did some googling and it wasn't obvious where I could find the jedis snapshots. Could you please point me in the right direction? 

Thanks

**Reply by sazzad16** [reply]
How do you use Jedis? Maven? Jar?

**Reply by jbank** [reply]
Maven

**Reply by sazzad16** [reply]
It's right in Jedis homepage here https://github.com/redis/jedis.

You'll need repository

```
    <repository>
      <id>snapshots-repo</id>
      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
    </repository>
```
 and dependency

```
    <dependency>
      <groupId>redis.clients</groupId>
      <artifactId>jedis</artifactId>
      <version>4.0.0-SNAPSHOT</version>
    </dependency>
```

**Reply by jbank** [reply]
🤦 sorry I should have seen that. Thanks again for answering my questions!

**Reply by sazzad16** [reply]
Great! Glad to help!
